packages/@okta/vuepress-site/docs/guides/sign-into-spa/main/index.md

```diff
@@ -24,10 +24,7 @@ This guide shows you how to use Okta as the user store for your single-page appl
 
```

Comment:
Also, should we move the stack-selector in the #define-a-callback-route to above the nutrition facts?